draft-ietf-rtcweb-jsep.xml

```diff
@@ -1178,8 +1178,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             MUST be used. Note that the remote endpoint may use a Lite
             implementation; implementations MUST properly handle remote
             endpoints which do ICE-Lite.</t>
-            <t>DTLS-SRTP, as specified in <xref target="RFC5763"></xref>,
-            MUST be used.</t>
+            <t>DTLS <xref target="RFC6347"/> or DTLS-SRTP <xref target="RFC5763"></xref>,
+            MUST be used, as specified in <xref target="I-D.ietf-rtcweb-security-arch"/></t>
```

Comment:
might want to say "as appropriate for the media type" or some such, but up to you